index.html

```diff
@@ -103,99 +114,108 @@ <h2 class="card-header">Multi-Post Stories</h2>
-                        </li>
-                        <li class="icon ">
-                            <a href="https://github.com/ibr5500 " target="”_blank”" rel="noreferrer noopener "><img src="./assets/github.svg " alt="Github icon " /></a>
-                        </li>
-                        <li class="icon ">
-                            <a href="# " rel="noreferrer "><img src="./assets/angellist.svg " alt="Angellist icon " /></a>
-                        </li>
-                    </ul>
+                <div class="about-continent">
+                    <h2 class="about-header">About <br/> Myself</h2>
+                    <div class="about-text">
+                        <p>
+                            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                        </p>
+                        <p class="contact">LET’S CONNECT</p>
+                    </div>
                 </div>
-                <div>
-                    <div class="about-button-container flex">
-                        <button type="button" class="button flex">Get my resume</button>
+                <div class="about-continent">
+                    <div>
+                        <ul class="about-icons flex list-style">
+                            <li class="icon ">
+                                <a href="https://twitter.com/ibr_ahmat " target="”_blank”" rel="noreferrer noopener "><img src="./assets/twitter.svg " alt="Twitter icon " /></a>
+                            </li>
+                            <li class="icon ">
+                                <a href="https://www.linkedin.com/in/ibrahim-ahmat-b5513b1a6/ " target="”_blank”" rel="noreferrer noopener "><img src="./assets/linkedin.svg " alt="Linked-In icon " /></a>
+                            </li>
+                            <li class="icon ">
+                                <a href="# " rel="noreferrer "><img src="./assets/medium.svg " alt="Medium icon " /></a>
+                            </li>
+                            <li class="icon ">
+                                <a href="https://github.com/ibr5500 " target="”_blank”" rel="noreferrer noopener "><img src="./assets/github.svg " alt="Github icon " /></a>
+                            </li>
+                            <li class="icon ">
+                                <a href="# " rel="noreferrer "><img src="./assets/angellist.svg " alt="Angellist icon " /></a>
+                            </li>
+                        </ul>
+                    </div>
+                    <div>
+                        <div class="about-button-container flex">
+                            <button type="button" class="button flex">Get my resume</button>
+                        </div>
```

Comment:
- [ ] As requested by the previous reviewer, Please use the correct gap, margins between your headings and down arrow icons and make sure it matches the design


### Your design

![image](https://user-images.githubusercontent.com/83803414/159944186-c3bdfa88-7561-4e49-b9d6-48eae381c499.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/159944235-cc75446f-476c-4d03-911f-77a12e44e993.png)